app/views/contacts/index.slim

```diff
@@ -0,0 +1,38 @@
+.container.our-team
+  .row
+    .col-sm-12.col-lg-12
+      h1
+        |  Our team
+      div.photos-container
```

Comment:
you don't to add `div` here. It is already converts slim `.photos-container` to html `<div class="photos-container"></div>`